app/controllers/time_summary_controller.rb

```diff
@@ -163,12 +174,17 @@ def time_summary_data_csv
         TimeConsumption.
           where(user: user).
           where(date: @start_date..@end_date).each do |tc|
-            time_consumptions[tc.date] = [
+            data = [
               user.full_name,
               tc.date,
               tc.resource.to_s,
-              helpers.number_with_precision(tc.amount, precision: 1)
+              helpers.number_with_precision(tc.amount, precision: 1),
+              (tc.resource.plan_item.business_unit.business_unit_type.name if tc.review?),
```

Comment:
¿Esto cuando trae nil no hace lío?

Digo cuando tienen detalle y de los dos tipos mezclados, porque me suena a que necesita esto:
```suggestion
              (tc.resource.plan_item.business_unit.business_unit_type.name if tc.review?).to_s,
```